datafusion/core/tests/dataframe/dataframe_functions.rs

```diff
@@ -750,3 +756,48 @@ async fn test_fn_upper() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_fn_encode() -> Result<()> {
```

Comment:
these functions weren't previously tested. I added tests to show that it is still possible to use the fluent API even with the moving of the `expr_fn` implementation